test/functorch/test_control_flow.py

```diff
@@ -1562,6 +1562,7 @@ def test_scan_dtype(self, reverse, compile_mode, device, dtype):
     )
     def test_associative_scan_dim(self, combine_mode, reverse, device):
         import random
+        random.seed(10)
```

Comment:
locally this "fixed" the test for me that broke previously (it was flaky, issue filed here: https://github.com/pytorch/pytorch/issues/137943)